### ISSUE TITLE
Support for Asian characters?

### ISSUE THREAD

**Issue by dormerod** [Kashomon/gpub#17]
Currently GPub doesn't seem to be able to properly output Asian characters which are written in an SGF.

This isn't a high priority feature right now (Asian characters are not essential for the books we're working on), but it doesn't seem entirely unexpected that someone would want to print characters in their English language go book.

**Comment by dormerod** [comment]
I suppose there is a larger and longer term issue to do with internationalization here. Presumably if GPub becomes popular, people will want to write go books in other languages and there would have to be translations and an option for choosing the language pack.

**Comment by dormerod** [comment]
I also noted a very minor, but related, issue when typing coup de grâce in a commentary and seeing the diacritic cause problems.

**Comment by Kashomon** [comment]
Yes, the achilles heal of the way I'm printing the books is that they require LaTeX, which doesn't support unicode in nice ways.
